Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -820,9 +821,83 @@ static RefPtr<CSSPrimitiveValue> consumeFontVariantCaps(CSSParserTokenRange& ran
         CSSValueUnicase, CSSValueTitlingCaps>(range);
 }
 
-static RefPtr<CSSPrimitiveValue> consumeFontVariantAlternates(CSSParserTokenRange& range)
+static RefPtr<CSSValue> consumeFontVariantAlternates(CSSParserTokenRange& range)
 {
-    return consumeIdent<CSSValueNormal, CSSValueHistoricalForms>(range);
+    // return early if nothing to parse
+    if (range.atEnd())
+        return nullptr;
+
+        // "normal" has to be single value.
```

Comment:
I suggest deleting this comment. It doesn't provide any more information beyond what the code directly below it provides.